CXDurationPicker/CXDurationPickerDayView.h

```diff
@@ -29,6 +29,8 @@
 
 // Colors
 //
+@property (nonatomic)BOOL roundedTermianls;
```

Comment:
Could you please address spelling? Should be "roundedTerminals" in all locations.